example/compose/nginx/backend/docker-compose.yml

```diff
@@ -1,11 +1,17 @@
-version: "3"
+version: '3'
 services:
   nginx-backend:
     build:
       context: example/compose/nginx/backend
     labels:
-      - traefik.frontend.rule=Host:home.example.com,public.example.com,secure.example.com,admin.example.com,singlefactor.example.com
-      - traefik.frontend.auth.forward.address=http://authelia-backend:9091/api/verify?rd=https://login.example.com:8080/
-      - traefik.frontend.auth.forward.tls.insecureSkipVerify=true
+      - 'traefik.frontend.rule=Host:home.example.com,public.example.com,secure.example.com,admin.example.com,singlefactor.example.com'
+      - 'traefik.frontend.auth.forward.address=http://authelia-backend:9091/api/verify?rd=https://login.example.com:8080/'
+      - 'traefik.frontend.auth.forward.tls.insecureSkipVerify=true'
+      - 'traefik.http.routers.protectedapps.rule=Host(`home.example.com`, `public.example.com`, `secure.example.com`, `admin.example.com`, `singlefactor.example.com`)'
+      - 'traefik.http.routers.protectedapps.entrypoints=https'
+      - 'traefik.http.routers.protectedapps.tls=true'
+      - 'traefik.http.routers.protecteapps.middlewares=authelia'
+      - 'traefik.http.middlewares.authelia.forwardAuth.address=http://authelia-backend:9091/api/verify?rd=https://login.example.com:8080/'
```

Comment:
Since `traefik.frontend.auth.forward.address` and `traefik.http.middlewares.authelia.forwardAuth.address` have the same value, I assume one is for Traefik v1 and for v2, am I correct?

If yes, that would be great to add a comment to say which one is v1 and v2.

Reply:
That's correct, frontends are Traefik 1.x, and middleware are Traefik 2.x.
I'll commend accordingly.